CHANGELOG.md

```diff
@@ -10,6 +10,20 @@
 
 ### Features
 
+- Add RNSentrySDK APIs support to @sentry/react-native/expo plugin ([#4633](https://github.com/getsentry/sentry-react-native/pull/4633))
+
+  This feature is opt-out, to enable it set `useNativeInit` to `true` in your `@sentry/react-native/expo` plugin configuration.
```

Comment:
What's the reason for making this opt-out?

This could break user apps if they don't have the sentry options JSON in their project; the added init would cause a runtime error.  

I think this should be opt-in first.

Reply:
@krystofwoldrich You are right! I think I've misread the [issue](https://github.com/getsentry/sentry-react-native/issues/4625) 😓

Reply:
Updated with adc81a54d475203b1c69e92855a5e7631339cb5f. I also removed the note from the changelog since it is opt-in first. I can describe how to opt-out on a docs PR.